2.cellprofiler_analysis/NF1_analysis_3channel.cppipe

```diff
@@ -239,7 +250,7 @@ ExportToDatabase:[module_num:17|svn_version:'Unknown'|variable_revision_number:2
     Database host:
     Username:
     Password:
-    Name the SQLite database file:NF1_data.sqlite
+    Name the SQLite database file:nf1_analysis.sqlite
```

Comment:
I'm wondering if we can be more specific with this file name... Maybe too late to do this? Usually, we name the file with the plate name

Reply:
Right, it is definitely the prior practice to name the files based on the plate. But this name is hardcoded into the pipeline. I renamed the file with sequential order after running the plate before starting the next one. In this case, I am running all plates simultaneously, which means that the only way this works without the SQLite files messing with each other since they have to have the same name is by putting them in separate folders. 

I think in this method, having all the files in their own folders with the Plate names works since when they are processed with CytoTable and pycytominer, the outputs will contain the plate names. 

If we want to put these files on figshare in the future, then we should rename all of the files then.